subworkflows/local/analyse_MT.nf

```diff
@@ -15,15 +14,19 @@ workflow ANALYSE_MT {
         dict          // channel: [ genome.dict ]
```

Comment:
```suggestion
        sequence_dict          // channel: [ genome.dict ]
```

Reply:
If dict has been changed to sequence_dict on align_and_call_MT.nf, then it should be done here too